### ISSUE TITLE
bug setstate ?

### ISSUE THREAD

**Issue by frodestorhaug** [chingyawhao/material-ui-next-pickers#16]
When I leave a page with a datepicker I get the following error :
Can't call setState (or forceUpdate) on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method

```
../../node_modules/react/cjs/react.development.js.Component.setState | @ | react.development.js:270
-- | -- | --
  | DateFormatInput._this.closeCalendar | @ | datepicker.js:72
  | (anonymous) | @ | datepicker.js:105
```

Is this a bug ?

**Comment by Anshuman71** [comment]
@frodestorhaug  i too have the same issue

**Comment by chingyawhao** [comment]
This issue should be solved at version 0.0.14
Please open another issue if it happens again
